### PR TITLE
Fix Api tests using fetch-mock

### PR DESCRIPTION
**Pull request by anuroop15**
added tests to improve coverage using fetch-mock API

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,6 +90,7 @@
     "istanbul-instrumenter-loader": "^3.0.0",
     "jasmine": "^2.5.3",
     "jasmine-core": "*",
+    "fetch-mock": "5.13.1",
```
Comment (ratneshraval) [inline package.json]: Commit corresponding yarn.lock

**Comment by anuroop15** [comment]
@ratneshraval : addressed above issues. Let me know if any changes needed.

File: test/javascript/tests.webpack.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import Enzyme from 'enzyme'
 import 'babel-polyfill'
 import Adapter from 'enzyme-adapter-react-16'
 import Promise from 'es6-promise-promise'
+require('whatwg-fetch')
```
Comment (ratneshraval) [inline test/javascript/tests.webpack.js]: We should be consistent between usage of require vs import.

**Comment by lavanyareddyreddy** [comment]
@anuroop15  can you fix the conflicts.

**Pull request merged**